lib/plugins/bidirectionalData.js

```diff
@@ -75,9 +75,7 @@ function extractVariables(item) {
 
```

Comment:
CNR entry missing?

Reply:
The PR destination is another branch (the one that removes Legacy support). Final CNR entry should be included there with all the details

Reply:
Ok. NTC